x/pylons/migrations/v4/migration.go

```diff
@@ -0,0 +1,27 @@
+package v4
+
+import (
+	"github.com/Pylons-tech/pylons/x/pylons/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+// TODO: update logic
+
+func migrateParamsStore(ctx sdk.Context, paramstore paramtypes.Subspace) {
+	if paramstore.HasKeyTable() {
```

Comment:
why we need that ?

Reply:
It's just a draft version, it will be updated by logics